Resources/Prototypes/Recipes/Lathes/Packs/science.yml

```
@@ -76,6 +76,7 @@
   - FauxTileJungleAstroGrass
   - FauxTileAstroIce
   - FauxTileAstroSnow
+  - FauxTileMowedAstroSnow # Delta-V
```

nit: Just comment as `# DeltaV`

```suggestion
  - FauxTileMowedAstroSnow # DeltaV
```